jiant/preprocess.py

```diff
@@ -661,7 +665,7 @@ def add_task_label_vocab(vocab, task):
         return
     log.info("\tTask '%s': adding vocab namespace '%s'", task.name, namespace)
 
-    if isinstance(task, SequenceGenerationTask):
+    if isinstance(task, SequenceGenerationTask) and not isinstance(task, LanguageModelingTask):
```

Comment:
This change is because of bad code design before (I think SequenceGenerationTask was initially created for machine translation and only supported non-BERT models). I think SequenceGenerationTask should be refactored in a separate PR, so just adding this here for now.

Reply:
Reasonable, though this does raise a real conflict: The _normal_ LM task is a sequence generation task, but MLM isn't. It may be simpler to just make MLM inherit directly from some broad task class, rather than from LMTask.